frontend/src/Editor/Editor.jsx

```diff
@@ -784,7 +784,11 @@ class Editor extends React.Component {
               <div
                 className={`canvas-container align-items-center ${!showLeftSidebar && 'hide-sidebar'}`}
                 style={{ transform: `scale(${zoomLevel})` }}
-                onClick={() => this.switchSidebarTab(2)}
+                onClick={(e) => {
+                  if (e.target.className === 'real-canvas') {
```

Comment:
This is preventing me from adding components inside modal, as the inspector panel wont close:
![inspector-not-closing-inside-modal](https://user-images.githubusercontent.com/20333972/145961661-8608b21f-afab-4d18-b1ea-270a4ff3b735.gif)